### PR TITLE
rewrite message notification to pinax

### PR DESCRIPTION
**Pull request by petrus7**
Issue:
https://github.com/GeoNode/geonode/issues/5927

### CODE REVIEW

File: geonode/messaging/__init__.py [hunk 1]
```diff
@@ -33,6 +34,8 @@
 class MessagingAppConfig(NotificationsAppConfigBase):
     name = 'geonode.messaging'
 
+    NOTIFICATIONS = (("create_message", _("Message received"), _("New message received in one of your threads")),)
```
Comment (giohappy) [inline geonode/messaging/__init__.py]: please call this notification "message_recevied"

Comment (petrus7) [reply geonode/messaging/__init__.py]: Done

File: geonode/messaging/notifications.py [hunk 2]
```diff
@@ -17,43 +17,41 @@
...
-from geonode.celery_app import app
 from django.conf import settings
-from user_messages.models import Message, Thread
+from user_messages.models import Message
 from user_messages.signals import message_sent
 
+from geonode.notifications_helper import send_notification
+
+
+def message_sent_notification(**kwargs):
```
Comment (giohappy) [inline geonode/messaging/notifications.py]: please also rename this method to "message_received_notification"

File: geonode/messaging/__init__.py [hunk 3]
```diff
@@ -33,6 +34,8 @@
 class MessagingAppConfig(NotificationsAppConfigBase):
     name = 'geonode.messaging'
 
+    NOTIFICATIONS = (("message_recevied", _("Message received"), _("New message received in one of your threads")),)
```
Comment (abulojoshua1) [inline geonode/messaging/__init__.py]: @petrus7, Little spelling error here, I think you meant "message_received" not "message_recevied".

File: geonode/messaging/migrations/0001_initial.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.11 on 2020-03-27 14:40
+
+from django.db import migrations
+
+def create_notice_settings(apps, schema_editor):
+    """Its here only to trigger post_migration signal which will create messaging notification"""
+    pass
```
Comment (abulojoshua1) [inline geonode/messaging/migrations/0001_initial.py]: Did you intend to make this pass? To me, it looks like this has no effect when you run migrations.

File: geonode/messaging/models.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# ########################################################################
+
+# Its required to fire post_migrate
```
Comment (abulojoshua1) [inline geonode/messaging/models.py]: @petrus7, what do you mean by `Its required to fire post_migrate`?

Comment (petrus7) [reply geonode/messaging/models.py]: This is required to trigger post_migrate signal. That signal is not fired if models.py does not exist in the app

File: geonode/messaging/migrations/0001_initial.py (hunk 4 above)
Comment (petrus7) [reply geonode/messaging/migrations/0001_initial.py]: I was tired of not fired post_migrate signal so i make empty migration which has no effect on the DB. But probably its not necessary

Comment (giohappy) [reply geonode/messaging/migrations/0001_initial.py]: If this sin't required please remove it. I don't like very much this kind of hacks @petrus7

**Comment by petrus7** [comment]
close because of this 
https://github.com/GeoNode/geonode/pull/5959

**Pull request closed without merging**